index.html

```diff
@@ -0,0 +1,136 @@
+                    </ul>
+                    <button id="cc-campaign" type="button">Book Now!</button>
+                    <div class="menu-end">
+                        <hr>
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <section id="headline">
+        <div class="flex-container">
+            <h2 class="headline__h2">"Hello! Aspiring Travellers"</h2>
+            <h1 class="headline__h1">Travellers Fest 2021</h1>
+            <div class="headline__message">
+                <p>​We are holding 2 day long travel fest in partnership with hilton hotels. During the two day period
+                    famous travellers will give talks on the value of travel in one's life and how to do it properly.
+                </p>
+            </div>
+            <h3 class="headline__date">2020.12.15(THU) ~ 16(FRI)</h3>
+            <h4 class="headline__location">@ Hilton Hotel Barcelona, Spain
+            </h4>
+        </div>
+    </section>
+    <section id="features">
+        <div class="flex-container">
+            <div class="features__head">
+                <h2>Features</h2>
+                <hr class="section__hr">
+            </div>
+            <div class="features__grid">
+                <div class="features__cards">
+                    <i class="fa fa-globe"></i>
+                    <h3>Around the World </h3>
+                    <p>Let the world be your oyester and jump into the most breathtaking
+                        experience of your life.</p>
+                </div>
+                <div class="features__cards">
+                    <i class="fa fa-tree"></i>
+                    <h3>Sceneic Outdoors</h3>
+                    <p>Outdoors is where life happens, so pack your bag for the best nature tours you have ever
+                        experienced.</p>
+                </div>
+                <div class="features__cards">
+                    <i class="fa fa-bed"></i>
+                    <h3>Quality Dwellings</h3>
+                    <p>Best in class hotels at your disposal during your tours with us , we pride ourselves in being
+                        able to deliver on our promises.
+                    </p>
+                </div>
+                <div class="features__cards">
+                    <i class="fa fa-camera-retro"></i>
+                    <h3>Quality Photos</h3>
+                    <p>You will get professional Photography as part of every package to make sure your best moments are
+                        captured beutifully forever.</p>
+                </div>
+                <div class="features__cards">
+                    <i class="fa fa-users"></i>
+                    <h3>World Class Tour Guides</h3>
+                    <p>We have the most seasoned and experienced tour guides who know the locations inside out along
+                        with all the historic contexts.</p>
+                </div>
+            </div>
```

Comment:
## [OPTIONAL]
- [ ] If we just check the original design we will find out that there is a hover effect on each feature card. I think kindly we can improve this design by adding this effect.
![image](https://user-images.githubusercontent.com/65068771/148632527-ac785523-d80e-4048-929e-82d6db218856.png)